### PR TITLE
[BEAM-1055] Display Data keys on Python are inconsistent

### PR DESCRIPTION
**Pull request by pabloem**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [x] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [x] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by pabloem** [comment]
r: @aaltay

File: sdks/python/apache_beam/io/textio.py [hunk 1]
```diff
@@ -296,12 +292,15 @@ def __init__(
...
+                           append_trailing_newlines, num_shards,
+                           shard_name_template, coder, compression_type)
 
   def apply(self, pcoll):
     return pcoll | Write(self._sink)
 
   def display_data(self):
-    return {'sink_dd': self._sink}
+    return {'sink_dd': self._sink,
+            'append_nwln': DisplayDataItem(self._append_trailing_newlines,
```
Comment (aaltay) [inline sdks/python/apache_beam/io/textio.py]: Maybe spell out this `append_newline` ?

**Comment by aaltay** [comment]
Thank you @pabloem. LGTM except for a minor comment.

**Comment by pabloem** [comment]
@bjchambers can you please merge? : )

**Comment by bjchambers** [comment]
Can you rebase (squashing commits) and add a more descriptive commit message. This describes a problem, but doesn't state that it was solved, or in what way things changed.

**Comment by pabloem** [comment]
Commits squashed. Jenkins build passing too.

**Comment by pabloem** [comment]
@bjchambers can you merge please? : )

**Comment by pabloem** [comment]
I just rebased to the most recent commit. Hopefully you're able to run `mvn verify` now (I was able : )

**Comment by pabloem** [comment]
The failed Jenkins run is due to a Java example failure. The Python SDK test passed.

**Comment by pabloem** [comment]
Relaunching tests.

**Comment by pabloem** [comment]
@bjchambers this is ready : )

**Pull request closed without merging**